README.md

```diff
@@ -42,14 +42,14 @@ The IBM Cloud Secrets Manager Java SDK allows developers to programmatically int
 <dependency>
     <groupId>com.ibm.cloud</groupId>
     <artifactId>secrets-manager</artifactId>
-    <version>2.0.15</version>
+    <version>2.0.16</version>
 </dependency>
 ```
 
 ##### Gradle
 
 ```gradle
-'com.ibm.cloud:secrets-manager:2.0.15'
+'com.ibm.cloud:secrets-manager:2.0.16'
```

Comment:
This change should be removed. Publication automation bumps version automatically.